dags/bqetl_experiments.py

```diff
@@ -0,0 +1,45 @@
+# Generated via https://github.com/mozilla/bigquery-etl/blob/master/bigquery_etl/query_scheduling/generate_airflow_dags.py
+
+from airflow import DAG
+from airflow.operators.sensors import ExternalTaskSensor
+import datetime
+from utils.gcp import bigquery_etl_query
+
+default_args = {
+    "owner": "ssuh@mozilla.com",
+    "start_date": datetime.datetime(2018, 11, 27, 0, 0),
+    "email": ["telemetry-alerts@mozilla.com", "ssuh@mozilla.com"],
+    "depends_on_past": False,
+    "retry_delay": datetime.timedelta(seconds=1800),
+    "email_on_failure": True,
+    "email_on_retry": True,
+    "retries": 2,
+}
+
+with DAG(
+    "bqetl_experiments", default_args=default_args, schedule_interval="0 1 * * *"
```

Comment:
I'm not sure if `bqetl_experiments` is a too general of a name for this DAG. Currently, it has only 1 task, but it's possible that more experiment related queries get added in the future.

Reply:
Maybe `bqetl_experiments_daily`?